packages/apps/shopify-app-remix/src/server/authenticate/admin/scope/__tests__/request.test.ts

```diff
@@ -86,6 +87,7 @@ it('when the shop is invalid an error is thrown', async () => {
     cookieName: SESSION_COOKIE_NAME,
     cookieValue: session.id,
   });
+  await mockGraphqlRequest()(200, responses.WITH_GRANTED_AND_DECLARED);
```

Comment:
but on the server side?, because I think that `admin.graphql` is not validating the store domain. This is the [only validation](https://github.com/Shopify/shopify-app-js/commit/326520cea916a3b52a4da540e0f0aa54fa378c94) I have found when the admin client is initialized. But this validation is different form the [sanitizeShop](https://github.com/Shopify/shopify-app-js/commit/a638cfe6ff82a23ebb9b767adba9c4a442d6e09e) used by the [redirectToInstallPage](https://github.com/Shopify/shopify-app-js/commit/06793fbf1d85e6a71f23c4ee907df8f5fc911ae2)

In fact the test is working because the request to the server is mocked. In a production environment it will return an error but I don't know if it does make sense to mock the response returning the server error.

Or maybe should we change the admin client initializer to use the `sanitizeShop`?